src/graph_coalescence/graph_coalescer.py

```diff
@@ -117,36 +115,43 @@ def coalesce_by_graph(opportunities, predicates_to_exclude=None, coalesce_thresh
                 threshold = len(nodes)
             if i >= threshold:
                 break
-            # for the first 78 enriched result,
+
             # Extract the pvalue and the set of chemical nodes that mapped the enriched link tuples
             best_enrich_p = link[0]
             best_grouping = link[7]
-            # I don't think this is right
-            # best_enrichments = list(filter(lambda x: (x[0] == best_enrich_p) and x[7] == best_grouping,enriched_links))
-            best_enrichments = [link]  # ?
-            # print(best_enrichments)
+            # Port forwarding generates: 'biolink:ameliorates'
```

Comment:
This is just because the remote redis is wrong, correct?  Let's have the code expect it to be right.

Reply:
Ok